### PR TITLE
dbt support

### PR DESCRIPTION
**Pull request by JakubMatejka**
https://keboola.atlassian.net/browse/KAC-267

### CODE REVIEW

File: cli/templates/tests/index.md [hunk 1]
```diff
@@ -129,10 +129,12 @@ The file supports complex values and can even refer to an OAuth configuration:
 }
 ```
 
+## Sensitive Values
+
```
Comment (JakubMatejka) [inline cli/templates/tests/index.md:133]: Tady jen fix že ty palceholdery pro env proměnný v testech templates můžou být použitý všude, nejen v inputs.json.

File: cli/commands/dbt/generate/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: Generate Command
+permalink: /cli/commands/dbt/generate/
+---
+
+* TOC
+{:toc}
+
+**Work with dbt inside your repository.**
+
+The commands must be run in a directory with a dbt project (i.e. containing `dbt_project.yml`).
```
Comment (michaljurecko) [inline cli/commands/dbt/generate/index.md:11]: drobnost: alebo v sub-directory

File: cli/templates/tests/index.md [hunk 3]
```diff
@@ -129,10 +129,12 @@ The file supports complex values and can even refer to an OAuth configuration:
...
 Sensitive values can be stored in environmental variables and referred to using a placeholder encapsulated by `##`. 
 The variables need to be prefixed by `KBC_SECRET_`.
```
Comment (michaljurecko) [inline cli/templates/tests/index.md:135]: Tu by bolo dobre nejak vysvetlit, ze ide pouzit aj non-secret ENVs a ze tie nie su obmedzene.
Tj. mohla by byt nejaka sekcia napr. `Placeholders` a `Sensitive Values` by bol iba pod-nadpis.

+ hodil by sa tu 2x link na to, ako nastavit ENVs v Github Actions a ako secrets (pre Moniku).

Comment (JakubMatejka) [reply cli/templates/tests/index.md]: Třeba tak? https://github.com/keboola/developers-docs/pull/225/commits/e7b8b3a911d993939ba8fc0fcb61d5c6f5706d7b

Comment (michaljurecko) [reply cli/templates/tests/index.md]: super :+1:

**Review by michaljurecko — APPROVED** [review]

File: cli/commands/dbt/generate/index.md (hunk 2 above)
Comment (JakubMatejka) [reply cli/commands/dbt/generate/index.md]: https://github.com/keboola/developers-docs/pull/225/commits/3b45cac319c8662f701c41f1fd463292946812ca

**Review by hhanova — APPROVED** [review]

**Pull request merged**